README.asciidoc

```diff
@@ -444,6 +447,24 @@ Leaving it blank has the same effect.
 
 |===
 
+For convenience, here is a snippet to paste into your theme/config:
+
+----
+face global InfoDefault               Information
+face global InfoHeader                Information
+face global InfoBlock                 Information
```

Comment:
I'd rather keep `InfoBlock`. Since we can't provide full code syntax highlighting, this gives at least some way to distinguish code blocks from regular text.

Reply:
Makes sense. BTW, I sketched some ideas for experimenting with a scratch buffer for hover in #514.
It looks like it has the same problem - it's not so easy to extract language-specific highlighting from Kakoune.
Unless we just give the buffer the markdown filetype..